### PR TITLE
use weekday fn from days.js to respect locale weekday reordering

### PR DESCRIPTION
**Pull request by flo-l**
eg. MO-SO instead of SO-SA

### CODE REVIEW

**Comment by flo-l** [comment]
The tests are green on my box.

File: src/utils/LocaleUtil.ts [hunk 1]
```diff
@@ -3,9 +3,11 @@ import { IDatePicker } from '../common/@types';
 import { range } from '../utils/ArrayUtil';
 import * as localeData from 'dayjs/plugin/localeData';
 import * as localizedFormat from 'dayjs/plugin/localizedFormat';
+import * as weekday from 'dayjs/plugin/weekday';
 
 dayjs.extend(localeData);
 dayjs.extend(localizedFormat);
+dayjs.extend(weekday)
```
Comment (y0c) [inline src/utils/LocaleUtil.ts]:
```suggestion
dayjs.extend(weekday);
```
it's lint error, missing semicolon

Comment (flo-l) [reply src/utils/LocaleUtil.ts]: oh thank you very much. I'll fix it asap!

**Comment by flo-l** [comment]
I had to bump the dayjs dependency to get the weekday functionality.

The current version in the package is 1.8.12. Weekday came one release later, in [1.8.13](https://github.com/iamkun/dayjs/blob/dev/CHANGELOG.md#1813-2019-04-26). I updated to the latest version: 1.8.21. However, I can change that if you prefer.

**Comment by flo-l** [comment]
hmm I can't run yarn install locally. I get an error during gyp compilation. @y0c could you please update the yarn lockfile?

**Comment by y0c** [comment]
@flo-l I'll try tomorrow or the day after tomorrow.

**Comment by y0c** [comment]
@flo-l I update yarn lockfile to dev branch.

**Comment by flo-l** [comment]
thank you!

I'm afraid I can't build the current master on my machine. I get compile errors during yarn install. I'm on Fedora Linux with Node 12.15.

I can get the build to pass if I bump node-sass from 4.11.0 to 4.12.0.

**Comment by flo-l** [comment]
> thank you!
> 
> I'm afraid I can't build the current master on my machine. I get compile errors during yarn install. I'm on Fedora Linux with Node 12.15.
> 
> I can get the build to pass if I bump node-sass from 4.11.0 to 4.12.0.

Well I'm an idiot and didn't pull from your repo, but from my fork. It works now.

I changed the type of Locale from any to string | ILocale. If that is not what you intended I'll revert this change!

File: package.json [hunk 2]
```diff
@@ -112,7 +112,7 @@
     "husky": "^1.3.1",
     "jest": "^24.7.0",
     "lint-staged": "^8.1.0",
-    "node-sass": "4.11.0",
+    "node-sass": "4.12.0",
```
Comment (y0c) [inline package.json]: you fell out `dayjs` deps while resolving the conflict. set 1.8.21

**Comment by y0c** [comment]
Well done. Thank you.

**Comment by flo-l** [comment]
> Well done. Thank you.

Thank you too for your lib and the merge :)

**Comment by flo-l** [comment]
One last request: Could you please do a release so I can use upstream in my project?

**Comment by y0c** [comment]
@flo-l update version 1.0.3

**Pull request merged**